docs/quickstart.rst

```diff
@@ -400,6 +400,26 @@ First, set a password for the postgres user.  For this guide, we will use ``secu
     select now();
     \q
 
+Postgres on GCP
+---------------
+
+If you are deploying Security Monkey on GCP and decide to use Cloud SQL, it's recommended to run `Cloud SQL Proxy <https://cloud.google.com/sql/docs/postgres/sql-proxy>`_ to connect to Postgres. You'll need to run Cloud SQL Proxy on whichever machine needs to access Postgres, e.g. on your local workstation as well as on the GCE instance where you're running Security Monkey.
```

Comment:
I suggest switching the Proxy and Instance creation paragraphs  Your first instance is fine, but I would immediately follow it with creating the instance, then details about the proxy.